Weights/todo.md

```diff
@@ -0,0 +1,4 @@
+Later:
+* Cleanup tests.
+* Comment the code.
+* Check memory leaking.
```

Comment:
This file cannot be accepted in CGAL.

It will have to be removed before the PR can be merged. We still have plenty of time for that.

Reply:
Actually the feature page should have a draft of the modification to CHANGES, [in the appropriate section of the feature page](https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Weight_interface#CHANGES.md), so that reviewers can have a say.

Reply:
I removed all todos from my PRs. And I move this discussion to this [issue](https://github.com/CGAL/cgal/issues/5817).